README.md

```diff
@@ -17,17 +17,17 @@ Install viam-cartographer:
 
 * Linux aarch64:
     ```bash
-    sudo curl -o /usr/local/bin/carto_grpc_server http://packages.viam.com/apps/slam-servers/carto_grpc_server-stable-aarch64.AppImage
-    sudo chmod a+rx /usr/local/bin/carto_grpc_server
+    sudo curl -o /usr/local/bin/cartographer-module https://storage.googleapis.com/packages.viam.com/apps/slam-servers/cartographer-module-stable-aarch64.AppImage
```

Comment:
Hmm this seems off - there is both storage.googleapis.com and packages.viam.com. Should it remain just packages.viam.com?

Reply:
Yes it should stay package.viam

Reply:
Oh interesting, I copied/pasted this from the docs. Does this mean we should update the docs?

Reply:
oh, yes! i'm surprised these links are considered valid by the browser...

Reply:
ok fixed!

Reply:
thank you!